nodeJS/osc/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "interactops",
+  "version": "1.0.0",
+  "main": "index.js",
+  "license": "MIT",
+  "scripts": {
+    "dev": "node server.js"
+  },
+  "dependencies": {
+    "express": "^4.17.1",
+    "node-osc": "^5.2.1",
+    "socket.io": "^3.0.1",
+    "bower": "^1.4.1"
```

Comment:
shouldn't bower be a devDependency?

Reply:
I don't really know in which purpose bower package was added to the example I was relying on to build mine, and it seems that it is still working fine without it. Last commit remove this package